moVirt/src/main/java/org/ovirt/mobile/movirt/model/DataCenter.java

```diff
@@ -21,9 +21,33 @@ public Uri getBaseUri() {
         return CONTENT_URI;
     }
 
+    public enum Status {
+        UNKNOWN(-1),
+        UNINITIALIZED(0),
+        UP(1),
+        MAINTENANCE(2),
+        NOT_OPERATIONAL(3),
+        NON_RESPONSIVE(4),
```

Comment:
the valid data center statuses are this:
    UNINITIALIZED,
    UP,
    MAINTENANCE,
    NOT_OPERATIONAL,
    PROBLEMATIC,
    CONTEND;